tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -22,6 +22,8 @@ def parse_arguments():
     parser.add_argument("--commit_id", required=True, help="The last commit id included in this package.")
     parser.add_argument("--is_release_build", required=False, default=None, type=str,
                         help="Flag indicating if the build is a release build. Accepted values: true/false.")
+    parser.add_argument("--linux_build", required=False, default=None, type=str,
+                        help="specify the libonnxruntime.so lib values")
 
```

Comment:
Seems like you're trying to put two different signals into one parameter. It may be better to have a flag for including the linux so, and a separate one for specifying the location of that file if needed.